src/components/item/table.js

```diff
@@ -71,9 +71,9 @@ class ItemTable extends ItemIterator {
   }
 
   get classes() {
-    return ['table-body', {
+    return [super.classes, 'table-body', {
       'drop-target': !this.props.isDisabled,
-      'over': this.props.isOver
+      'dragging': this.state.isDragging
```

Comment:
Yes, just add both `drop-target` and `dragging` to the `ItemIterator` since we use it everywhere now.

Reply:
done